v1.1/datareference/datacite.xml

```diff
@@ -2,82 +2,688 @@
 <?xml-stylesheet type="text/xsl" href="https://terminologies.g-node.org/_resources/odmlDocument.xsl" xmlns:odml="http://www.g-node.org/odml"?>
 <odML version="1.1">
     <repository>https://terminologies.g-node.org/v1.1/terminologies.xml</repository>
-    <version>1.0</version>
-    <date>2019-03-26</date>
-    <!-- *********************************************************        -->
-    <!--                      Data reference section                      -->
-    <!--  The minimal description of a data reference is a URI linking back
-          to the repository the odml file or the data the odml document describes
-          resides in.
-          Further details common to the different types of data reference sections
-          can be added via the reference description section              -->
-    <!-- *********************************************************        -->
+    <version>1.1</version>
+    <date>2019-10-15</date>
+
     <section>
-        <type>DataReference</type>
         <name>DataCite</name>
-        <definition>A published dataset referenced by a DOI and registered with DataCite. The referenced dataset will contain either the odML file itself or the data the odML is annotating.
-        </definition>
+        <type>data_reference</type>
```

Comment:
```<type>datacite</type>?```

Reply:
Since we have datareferences besides datacite I would leave the top level type as `data_reference` for now to enable a broader search for these kind of sections, but we probably revisit this topic with a broader audience.